app/helpers/pagination_helper.rb

```diff
@@ -93,10 +94,10 @@ def pagination_settings(paginator, options)
   def per_page_links(paginator, options)
     Setting.per_page_options_array.inject('') do |html, n|
       if n == paginator.per_page
-        html + content_tag(:li, n, class: 'pagination--item -current')
+        html + content_tag(:li, n, class: 'op-pagination--item op-pagination--item_current')
```

Comment:
I was not sure about the best way to achieve that `op-pagination--item_current` has the same stylings as the base class. Should the base class be added (like I did it here) or should `op-pagination--item_current` inherit the styles in .sass-file ?

Reply:
Always add the base class. Some reasons why:

1. Semantically it is an `--item`, so it should get the class. This is also important when talking about elements. If you tell me you're doing something with `op-pagination--item`s, then I don't have to guess whether that includes modified items or not.
2. This makes it easier to select all `--item`s; otherwise you'd have to select every possible modifier
3. It makes it easier to read/write modifiers in sass, since they usually only have one or two rules, thus making it very clear what the modifier actually modifier

Reply:
👍